### ISSUE TITLE
Add category for fellowships/scholarships

### ISSUE THREAD

**Issue by trashbirdecology** [ogrants/ogrants#172]
Add `layout` argument or keyword to specify grant is fellowship/scholarship.  

I assume this argument could be used to help filter on the UI webpage at some point? 

Why? Allow end users to browse for and potentially identify new opportunities and not having to sift through non-fellowships.

**Comment by ethanwhite** [comment]
:+1: I definitely think better discovery is important going forward and I believe that's on @ha0ye's radar as the planning for the next phase of the project gets going.

**Comment by trashbirdecology** [comment]
Yeah sorry, I just cant help myself sometimes despite suspecting y'all had this on the radar
